src/main/java/frc/robot/Robot.java

```diff
@@ -44,7 +64,49 @@
   public void robotInit() {
     m_oi = new OI();
     // chooser.addOption("My Auto", new MyAutoCommand());
-    SmartDashboard.putData("Auto mode", m_chooser);
+    //SmartDashboard.putData("Auto mode", m_chooser);
+
+    /*
+    // hatchCamera = CameraServer.getInstance().startAutomaticCapture("hatch",0);
+    // hatchCamera.setResolution(640, 480);
+     portCamera = CameraServer.getInstance().startAutomaticCapture("port",1);
+     portCamera.setResolution(1280, 960);
```

Comment:
In my experience, you need to set camera parameters BEFORE starting automatic Capture, or the settings won't take effect.

Reply:
Yes I realize this is currently commented out.  Just an observation if you choose to bring it back